dbt_subprojects/dex/models/automated_trades/dex_mapping.sql

```diff
@@ -0,0 +1,475 @@
+	('mapping','uni-v2','polygon','0x5De74546d3B86C8Df7FEEc30253865e1149818C8','firebird'),
+	('mapping','uni-v2','fantom','0xc35DADB65012eC5796536bD9864eD8773aBc74C4','sushi'),
+	('mapping','uni-v2','fantom','0xEF45d134b73241eDa7703fa787148D9C9F4950b0','spiritswap'),
+	('mapping','uni-v2','fantom','0x733A9D1585f2d14c77b49d39BC7d7dd14CdA4aa5','paintswap'),
+	('mapping','uni-v2','fantom','0xB2435253C71FcA27bE41206EB2793E44e1Df6b6D','wakaswap'),
+	('mapping','uni-v2','fantom','0x991152411A7B5A14A8CF0cDDE8439435328070dF','hyperswap'),
+	('mapping','uni-v2','fantom','0xd9820a17053d6314B20642E465a84Bf01a3D64f5','dfyn'),
+	('mapping','uni-v2','fantom','0xeAcC845E4db0aB59A326513347a37ed4E999aBD8','shiba'),
+	('mapping','uni-v2','fantom','0x6178C3B21F7cA1adD84c16AD35452c85a85F5df4','zdex'),
+	('mapping','uni-v2','fantom','0xD9473A05b2edf4f614593bA5D1dBd3021d8e0Ebe','bomb'),
+	('mapping','uni-v2','fantom','0x3fAaB499b519fdC5819e3D7ed0C26111904cbc28','solidly'),
+	('mapping','uni-v2','avalanche','0xefa94DE7a4656D787667C749f7E1223D71E9FD88','pangolin'),
+	('mapping','uni-v2','avalanche','0x9Ad6C38BE94206cA50bb0d90783181662f0Cfa10','traderjoe'),
+	('mapping','uni-v2','avalanche','0xe0C1bb6DF4851feEEdc3E14Bd509FEAF428f7655','lydia'),
+	('mapping','uni-v2','avalanche','0x4Fe4D8b01A56706Bc6CaD26E8C59D0C7169976b3','olive'),
+	('mapping','uni-v2','avalanche','0x5C02e78A3969D0E64aa2CFA765ACc1d671914aC0','complus'),
+	('mapping','uni-v2','avalanche','0x58C8CD291Fa36130119E6dEb9E520fbb6AcA1c3a','yeti'),
+	('mapping','uni-v2','avalanche','0xCFBA329d49C24b70F3a8b9CC0853493d4645436b','canary'),
+	('mapping','uni-v2','avalanche','0x58A08bc28f3E8dab8Fb2773D8f243bC740398b09','party'),
+	('mapping','uni-v2','arbitrum','0xf1D7CC64Fb4452F05c498126312eBE29f30Fbcf9','uniswap v2'),
+	('mapping','uni-v2','arbitrum','0x359F20Ad0F42D75a5077e65F30274cABe6f4F01a','dxswap'),
+	('mapping','uni-v2','harmony','0x9014B937069918bd319f80e8B3BB4A2cf6FAA5F7','defikingdoms'),
+	('mapping','uni-v2','aurora','0xc66F594268041dB60507F00703b152492fb176E7','trisolaris'),
+	('mapping','uni-v2','okex','0xE7Fb3e833eFE5F9c441105EB65Ef8b261266423B','dfyn'),
+	('mapping','uni-v2','optimism','0x8BCeDD62DD46F1A76F8A1633d4f5B76e0CDa521E','zipswap'),
+	('mapping','uni-v2','gnosis','0xc35DADB65012eC5796536bD9864eD8773aBc74C4','sushiswap'),
+	('mapping','uni-v3','blast','0x1a8027625c830aac43ad82a3f7cd6d5fdce89d78','blasterswap'),
+	('mapping','uni-v3','monoswap','0x48d0f09710794313f33619c95147f34458bf7c3b','monoswap'),
+	('mapping','uni-v2','base','0xba06043a777652baf540ccc785edafd94ee05b37','archly')
+) as x (src, dex_type, blockchain, factory, project)
+
+-- for easier handling, you can download the csv file from dune and import it into a spreadsheet
+-- after having wokred with the spreadsheet and added the missing projects, you can export the spreadsheet again as csv
+-- and import it here again.
+-- sqlizer works great for converting the csv file into a sql table.
```

Comment:
```suggestion
```

Reply:
comment irrelevant now that we moved away from seed?

Reply:
nope still relevant, sqlizer is a tool that can convert from csv to these sql statements